### PR TITLE
Allow ftw.upgrade's control panel to work with jQuery 1.9+

### PR DESCRIPTION
**Pull request by neilferreira**
Fixes https://github.com/4teamwork/ftw.upgrade/issues/112

### CODE REVIEW

**Comment by jone** [comment]
Awesome, thanks!
Do you mind to add an entry in the `HISTORY.txt`?

File: setup.py [hunk 1]
```diff
@@ -61,6 +61,7 @@
         'requests',
         'setuptools',
         'tarjan',
+        'plone.app.jquery >= 1.9.1',
```
Comment (jone) [inline setup.py]: This breaks Plone 4.3 support.

Why did you choose `1.9.1`?
You are introducing the use of [`.on`](http://api.jquery.com/on/), which was introduced in jQuery 1.7, which was packaged in `plone.app.jquery = 1.7.2.1`, the version used for Plone `4.3`. Therefore I assume that this JavaScript will run with `plone.app.jquery = 1.7.2.1`.

Actually, I would prefer to not have a version constraint here at all.
The reason is that we sometimes use `ftw.upgrade` for upgrading very old installations.
Although we no longer guarantee compatibility with older Plone versions it might work anyway as long as we are not blocking compatibility by using version constraints.

I therefore propose to just remove this line. Do you agree?

Comment (neilferreira) [inline setup.py]: I agree with your point.  

I'll do this instead:
Remove the requirement of plone.app.jquery >= 1.9.1
Modify manage.js to raise an error if jQuery <= 1.7 is being used.

Comment (jone) [inline setup.py]: 👍 great!

**Comment by neilferreira** [comment]
Damn I was in a huge rush when I did this earlier to to able to show the previously run builds on a website of mine and actually broke some of the functionality in the process (uncheck all / check new).    Also realized .live wasn't actually needed in the first place, just using .click() would have been sufficient. 

Ignore the commits, just take a look at "Files changed" in the PR now.  Will squash these commits too.

**Comment by neilferreira** [comment]
Squashed.

tl;dr:
Updated usage of .live to simply using .click as elements were not cleated on the fly
Updated usage of .attr in favour of .prop

**Comment by jone** [comment]
Awesome, thank you! ✨ 👍

**Comment by jone** [comment]
Released as `ftw.upgrade = 2.0.1` https://pypi.python.org/pypi/ftw.upgrade/2.0.1
